mongoServer.js

```diff
@@ -2,47 +2,144 @@
 var express = require('express');
 var MongoClient = require('mongodb').MongoClient;
 var bodyParser = require('body-parser');
+var ObjectID = require('mongodb').ObjectID;
```

Comment:
Can you move all the changes made here into another file? I don't want to group to be overwhelmed by this much code on stage 2 of the workshop.